src/tests/Tests.hs

```diff
@@ -89,7 +90,11 @@ makeCompilerTests packageConf basePath rand = do
 fns :: String -> (String, String, FilePath)
 fns file =
   ( root
+#if TYPESCRIPT
```

Comment:
`toTsName`/`toJsName` could also be abstracted in the CPP module i mentioned above.